apps/admin/keyvault-flexvol/keyvault-flexvol.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    app: keyvault-flexvolume
```

Comment:
is this really still needed?

Reply:
yeah, saw a couple of jobs using it in prod. (we forgot about jobs when we chased people)

Reply:
Its there to add it only to prod